Library/Formula/yaml-cpp.rb

```diff
@@ -28,11 +28,7 @@ def install
     ENV.cxx11 if build.cxx11?
     ENV.universal_binary if build.universal?
     args = std_cmake_args
-    if build.with? "static-lib"
-      args << "-DBUILD_SHARED_LIBS=OFF"
-    else
-      args << "-DBUILD_SHARED_LIBS=ON"
-    end
+    args << "-DBUILD_SHARED_LIBS=" + (build.with?("static-lib") ? "OFF" : "ON")
```

Comment:
Let's leave this how it was. The current style is very established and new users find it nice and easy to understand. I've filed a PR to disable the Rubocop cop that flags this syntax.